submissions/BlueLamantine/friends app/js/main.js

```diff
@@ -0,0 +1,329 @@
+class Render {
+  constructor(results, info) {
+    this.data = results;
```

Comment:
Give less generic name to the variable - https://learn.javascript.ru/ninja-code

Reply:
Ok, got it!